build.gradle

```diff
@@ -119,7 +122,7 @@ compileJava {
 dependencies {
 
     // Jackson dependencies
-    def jacksonVersion = "2.7.9"
+    def jacksonVersion = "2.13.4"
```

Comment:
Why don't we use parent bom to define the dependencies' versions ?

Reply:
@nebilbenmabrouk I guess it was tried in the past but proved impossible. This is probably because connector-iaas depends on cloud SDKs on which parent bom will mess library versions